Modules/+Drivers/+msquared/EMM.m

```diff
@@ -107,6 +107,8 @@ function delete(obj)
             try  % Might not be a good IP, so try only
                 % Give control back to solstis ICE BLOC
                 obj.com('set_wavelength',800,false,'infrared');
+                pause(1);
```

Comment:
Should we use some other timeout property/pref?

Reply:
This is old interface.